.travis.yml

```diff
@@ -31,6 +31,10 @@ install:
   # Ensure Gradle wrapper is executable, download wrapper and show version
   - chmod +x ./gradlew; ls -l gradlew; ./gradlew wrapper -v
 
-script: ./gradlew clean build -Pbuild=dev
-script: ./gradlew test -Pbuild=dev
+script:
+  - ./gradlew clean build -Pbuild=dev jacocoTestReport
+  - ./gradlew test -Pbuild=dev
 # script: ./gradlew connectedAndroidTest -Pbuild=dev
+
+#after_success:
+#  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Why is this removed?

Reply:
Just in case it breaks the build since I'm not sure if CodeCov has been enabled yet

Reply:
You should be able to run this now.